packages/framer-motion/src/components/Reorder/__tests__/server.ssr.test.tsx

```diff
@@ -13,7 +13,7 @@ describe("Reorder", () => {
         const staticMarkup = renderToStaticMarkup(<Component />)
         const string = renderToString(<Component />)
 
-        const expectedMarkup = `<article><main draggable="false" style="z-index:unset;transform:none;-webkit-touch-callout:none;-webkit-user-select:none;user-select:none;touch-action:pan-x"></main></article>`
```

Comment:
This should only be the case if `originX` etc or `transformOrigin` have been explicitly set.